pandas/core/arrays/sparse.py

```diff
@@ -1735,6 +1735,9 @@ def __unicode__(self):
             fill=printing.pprint_thing(self.fill_value),
             index=printing.pprint_thing(self.sp_index))
 
+    def _formatter(self, boxed=False):
+        return None
```

Comment:
what does this mean / do?

Reply:
Falls back to the default for whatever `*Formatter` is printing the array. So this will end up being something like `FloatFormatter._value_formattter()`, or whatever, depending on the dtype. I'll add a comment to this (and clarify the one for categorical)

Reply:
Hopefully clarified in a7ef10406be10e21605e32b05fe196711f46bed4

Reply:
yep